mne/io/kit/kit.py

```diff
@@ -680,27 +684,19 @@ def get_kit_info(rawfile):
 
         # Creates a list of dicts of meg channels for raw.info
         logger.info('Setting channel info structure...')
-        ch_names = {}
-        ch_names['MEG'] = ['MEG %03d' % ch for ch
-                           in range(1, sqd['n_sens'] + 1)]
-        ch_names['MISC'] = ['MISC %03d' % ch for ch
-                            in range(1, sqd['nmiscchan'] + 1)]
```

Comment:
PyCharm suggested refactors and what I found clearer (more red than green :) ), but I can revert if you want, there is no change in function.